scripts/install.sh

```diff
@@ -329,7 +329,8 @@ function kurl_config() {
         --from-literal=upload_certs_expiration="$CERT_KEY_EXPIRY" \
         --from-literal=service_cidr="$SERVICE_CIDR" \
         --from-literal=pod_cidr="$POD_CIDR" \
-        --from-literal=kurl_install_directory="$KURL_INSTALL_DIRECTORY_FLAG"
+        --from-literal=kurl_install_directory="$KURL_INSTALL_DIRECTORY_FLAG" \
+        --from-literal=kurl_cis_compliance="$CIS_COMPLIANCE"
```

Comment:
see my comment here https://github.com/replicatedhq/ekco/pull/43#discussion_r830236527

Reply:
removed and added it to ekco configmap